sql/core/src/test/resources/sql-tests/results/ansi/string-functions.sql.out

```diff
@@ -14,7 +29,7 @@ select format_string()
 struct<>
 -- !query output
 org.apache.spark.sql.AnalysisException
-requirement failed: format_string() should take at least 1 argument; line 1 pos 7
+0; line 1 pos 7
```

Comment:
hmm, why is that? We should see similar error as for `select concat_ws()`.

Reply:
fixed